Block/Variation.php

```diff
@@ -114,8 +135,16 @@ public function getAbstractObject()
     {
         $store = $this->nostoHelperScope->getStore(true);
 
+        // We inject the active variation tag if the exchange rates are used or
+        // if the price variations are used and the active variation is the
+        // default one
         if ($this->nostoHelperCurrency->exchangeRatesInUse($store)
-            || $this->nostoHelperData->isPricingVariationEnabled($store)
+            || ($this->nostoHelperData->isPricingVariationEnabled($store)
+                && $this->nostoHelperVariation->isDefaultVariationCode(
+                    $this->nostoHelperCustomer->getGroupCode()
```

Comment:
Missing try catch

Reply:
We'll let it blow up. We want it to be obvious that the price variations cannot be used.